src/components/ReportCard/ReportCard.js

```diff
@@ -1,31 +1,132 @@
-import React, { Component } from 'react';
+import { Component } from 'react';
 import { Col, Container, Row, TabContent, TabPane, Nav, NavItem, NavLink } from "reactstrap";
 import '@devexpress/dx-react-grid-bootstrap4/dist/dx-react-grid-bootstrap4.css';
 import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
 import { faFile } from '@fortawesome/free-regular-svg-icons';
+import { connect } from 'react-redux';
+import { store } from '../../App';
+import { dataToTableConverter, experimentalDataConverter } from '../../helpers/dataHelper';
+import { Grid, Table, TableColumnResizing, TableHeaderRow } from '@devexpress/dx-react-grid-bootstrap4';
 
 class ReportCard extends Component {
     constructor(props) {
         super(props);
         this.state = {
-            activeTab: '1'
+            activeTab: '1',
+            summaryDataset: store.getState().summaryDatasets
         }
         this.setActiveTab = this.setActiveTab.bind(this);
     }
 
+    componentDidMount() {
```

Comment:
What is this? If we didn't get summary datasets, we are navigating the user back to the Atlas Home page? That seems odd.

Reply:
This is if we go to the /repository/report page without having selected a participant, it sends you back to the /repository page. Otherwise the page would just say "No Data"

Reply:
Ah, ok!